### PR TITLE
Use PUT when updating OAuth Server Item

### PR DESCRIPTION
**Pull request by s4nji**

### CODE REVIEW

File: src/modules/pages/OAuthServerPage/oAuthServerContainer.js [hunk 1]
```diff
@@ -19,5 +20,6 @@ export default connect(
         fetchOAuthServer,
         fetchOAuthServerSchema,
         saveOAuthServer,
```
Comment (ivanzusko) [inline src/modules/pages/OAuthServerPage/oAuthServerContainer.js]: plz remove this as far as we don't use it anymore

File: src/modules/pages/OAuthServerPage/oAuthServerContainer.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {
     fetchOAuthServer,
     fetchOAuthServerSchema,
     saveOAuthServer,
```
Comment (ivanzusko) [inline src/modules/pages/OAuthServerPage/oAuthServerContainer.js]: plz remove this as far as we don't use it anymore

**Comment by ivanzusko** [comment]
Awesome, @s4nji !
As an improvement, we can also change `isFetching` while saving or updating, just in case if it will take some time to proceed...

**Comment by s4nji** [comment]
@ivanzusko @rafaeljesus Updated, please re-check 🙂

**Pull request merged**